index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
Yeah, I've looked before, just couldn't tell off top of my head about `let` - and wanted to understand the general direction. Thanks for clarification.